src/HasParent.php

```diff
@@ -123,4 +123,16 @@ protected function getParentClass()
 
         return $parentClassName ?: $parentClassName = (new ReflectionClass($this))->getParentClass()->getName();
     }
+
+    /**
+     * Get the fillable attributes for the model.
+     *
+     * @return array
+     * @throws \ReflectionException
+     */
+    public function getFillable()
+    {
+        $parentFillable = (new ReflectionClass($this))->getParentClass()->newInstance()->getFillable();
```

Comment:
I think using the `getParentClass` is a bit better, that way you won't create an reflection class instance on every subsequent call to `getFillable` as this is cached by that method.
```suggestion
        $parentFillable = $this->getParentClass()->getFillable();
```

Reply:
I like this suggestion. Will definitely add it and append some tests